users.txt

```diff
@@ -1,3 +1,4 @@
+session-manager 9999
```

Comment:
Why not 10006?..

Reply:
10002+ are already used on our nodes. The biggest UID +1 is used for each new user.